tf-psa-crypto/tests/suites/test_suite_psa_crypto.function

```diff
@@ -1234,6 +1234,30 @@ static void interruptible_signverify_get_minmax_completes(uint32_t max_ops,
         *max_completes = 1;
     }
 }
+
+static uint32_t interruptible_key_agreement_get_min_num_ops(size_t key_bits)
+{
+    uint32_t min_values[3][5] = {
```

Comment:
What about curve families other than SECP_R1?

Reply:
It depends on the curve size, and we only support Short Weierstrass curves for interruptible operations.